dataactvalidator/scripts/loaderUtils.py

```diff
@@ -10,7 +10,7 @@ def clean_col_names(field):
 
 class LoaderUtils:
     @classmethod
-    def padFunction(self, field, padTo, keepNull):
+    def padFunction(cls, field, padTo, keepNull):
```

Comment:
do we even need `cls` here since it's not referenced at all? This could instead be a static method, no?